boards/arm/nucleo_l433rc/nucleo_l433rc.yaml

```diff
@@ -0,0 +1,16 @@
+identifier: nucleo_l433rc
+name: NUCLEO-L433RC-P
```

Comment:
This field is now used for automatic doc generation, it should match .rst title entry: ST Nucleo L433RC-P